src/components/collection-card/docs.mdx

```diff
@@ -28,15 +28,7 @@ With logos as heading:
     gap: '16px',
   }}
 >
-  {[
-    'docker',
-    'github',
-    'microsoft-azure',
-    'oci',
-    'google-cloud',
-    'terraform-cloud',
-    'aws',
-  ].map((logo, index) => {
+  {Object.keys(CompanyLogoOption).map((logo, index) => {
```

Comment:
Tangential to the PR but including cause I messed with `CompanyLogoOption` in the process of doing TypeScript pre-work for `heading` stuff: I've threaded through `CompanyLogoOption` as a named export of `collection-card`, so that we can iterate over the enum in Swingset. This way, if we ever miss an option, I _think_ we should see a failed build rather than pushing something broken to production.